app/src/main/java/org/pslab/sensors/SensorADS1115.java

```diff
@@ -188,7 +188,10 @@ private void setSensorDock() {
         playPauseButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                if (play) {
+                if (play & scienceLab.isConnected()) {
```

Comment:
@CloudyPadmal Is there a specific reason to use single & operator and not && , I think && should be  better.

Reply:
Ahh yes! I'll make the changes! :+1: